packages/back-end/src/services/auth/index.ts

```diff
@@ -248,6 +254,45 @@ export async function processJWT(
         dateCreated: new Date(),
       });
     };
+
+    if (IS_CLOUD) {
+      const build = getBuild();
+      req.gb = new GrowthBook({
+        apiHost: "https://cdn.growthbook.io",
+        clientKey: GB_SDK_ID,
+        enableDevMode: true,
+        stickyBucketService: new BrowserCookieStickyBucketService({
+          jsCookie: req.cookies,
+        }),
+        trackingCallback: () => {
+          // TODO: How to send event to Jitsu?
+        },
+        attributes: {
+          id: user.id,
+          url: req.originalUrl,
+          cloud: IS_CLOUD,
+          freeSeats: req.organization?.freeSeats,
+          discountCode: req.organization?.discountCode,
+          organizationId: req.organization?.id,
+          accountPlan: req.organization
+            ? getAccountPlan(req.organization)
+            : undefined,
+          superAdmin: user.superAdmin,
+          orgDateCreated: req.organization?.dateCreated,
+          anonymous_id: req.cookies["gb_device_id"],
```

Comment:
In the FE code this is from Jitsu cookie -- found this as an alternative but also it does not feel right.

Maybe we just don't include this `anonymous_id` as all requests from the back-end will be authenticated?